news/6004.removal

```diff
@@ -0,0 +1 @@
+Read version in setup.py without re
```

Comment:
I would mark this as trivial since it isn't really user-facing.

Reply:
Fixed.